moveit_ros/benchmarks/package.xml

```diff
@@ -1,32 +1,24 @@
 <package>
   <name>moveit_ros_benchmarks</name>
-  <version>0.6.6</version>
-  <description>MoveIt tools for benchmarking</description>
+  <version>0.0.1</version>
```

Comment:
should be same as https://github.com/ros-planning/moveit/blob/kinetic-devel/moveit/package.xml#L4